src/SupportUsBundle/Resources/views/Default/about.html.twig

```diff
@@ -46,6 +46,7 @@
             <a href="https://itunes.apple.com/app/qctools/id1234563987" target="_blank">QCTools</a>,
             <a href="https://itunes.apple.com/app/bwf-metaedit/id860703858" target="_blank">BWF MetaEdit</a>,
             <a href="https://itunes.apple.com/app/avi-metaedit/id1289778020" target="_blank">AVI MetaEdit</a>,
+            <a href="https://itunes.apple.com/app/mov-metaedit/1299543806" target="_blank">MOV MetaEdit</a>,
```

Comment:
This url do not respond

Reply:
It will tomorrow ;-)

Reply:
Removed until MOV MetaEdit MAS is ready.